src/source_sampling.cpp

```diff
@@ -179,6 +187,17 @@ void pyne::Sampler::setup() {
   }
   else throw std::invalid_argument("Mesh file must contain only tets or hexes.");
 
+  // Assign MeshMode: VOXEL, SUBVOXEL, TET
+  if (ve_type == moab::MBHEX){
+     if (mode < 3){
```

Comment:
Can we define a constant here instead of '3'?

Reply:
I defined it as a const int `SUBVOXEL_START` in the code.